modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/AbstractPythonCodegen.java

```diff
@@ -110,6 +110,9 @@ public AbstractPythonCodegen() {
                         "header_params", "form_params", "local_var_files", "body_params", "auth_settings",
                         // @property
                         "property",
+                        // pydantic keyword
```

Comment:
`schema` may be Pydantic-specific, but not the others I think:

```suggestion
                        // typing keywords
```

Note that we may have the same problem with any basic types (int, bool, datetime, etc.)